pyls/plugins/ctags.py

```diff
@@ -0,0 +1,251 @@
+# Copyright 2017 Palantir Technologies, Inc.
+import io
+import logging
+import os
+import re
+import subprocess
+
+
+from pyls import hookimpl, uris
+from pyls.lsp import SymbolKind
+
+log = logging.getLogger(__name__)
+
+DEFAULT_TAG_FILE = "${workspaceFolder}/.vscode/tags"
+DEFAULT_CTAGS_EXE = "ctags"
+
+TAG_RE = re.compile((
+    r'(?P<name>\w+)\t'
+    r'(?P<file>.*)\t'
+    r'/\^(?P<code>.*)\$/;"\t'
+    r'kind:(?P<type>\w+)\t'
+    r'line:(?P<line>\d+)$'
```

Comment:
Does not consider Windows line endings (\r\n). The regex will fail on Windows files. Suggest changing line to:

`r'line:(?P<line>\d+)\s*$'`